qa/L0_sequence_batcher/test.sh

```diff
@@ -183,6 +183,17 @@ export USE_SINGLE_BUFFER
 #   models4 - four instances with batch-size 1
 rm -fr *.log  models{0,1,2,4} queue_delay_models && mkdir models{0,1,2,4} queue_delay_models
 
+# Search BACKENDS to determine if a backend should be tested
+function should_test_backend() {
+  local target_backend=$1
```

Comment:
Sorry for the low quality comment but you could simplify this to the following 
```
if echo "${BACKENDS[@]}" | grep -q "${target_backend}"; then
```

Basically list the backends in one string, and assuming its a distinct set of names, grep for the target backend in the set

Reply:
This is neat! I modified the idea slightly to use the `-c` option.